### ISSUE TITLE
Setup shows the ERRO even fix function do needful

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/crc#188]
```
19:53:36 + crc setup
19:53:36 INFO[0000] Error loading global state: open /root/.crc/globalstate.json: no such file or directory 
19:53:36 crc - Local OpenShift 4.x cluster
19:53:36 INFO  Caching oc binary                           
19:53:36 ERRO Error occured: oc binary is not cached.      
19:53:38 INFO  Copying '/tmp/crc103903385/oc' to '/root/.crc/cache/oc/oc' 
19:53:38 INFO  Setting up virtualization                   
19:53:38 INFO  Setting up KVM                              
19:53:38 INFO  Installing Libvirt                          
19:53:38 INFO  Adding user to libvirt group                
19:53:38 ERRO Error occured: [root] not part of libvirtd group 
19:53:38 INFO  Enabling libvirt                            
19:53:38 INFO  Starting Libvirt service                    
19:53:38 INFO  Installing crc-driver-libvirt               
19:53:39 INFO  Setting up Libvirt crc network              
19:53:40 ERRO Error occured: Libvirt network crc not found 
19:53:40 INFO  Starting Libvirt crc network                
19:53:40 ERRO Error occured: Libvirt crc network is not active 
19:53:41 INFO  Writing Network Manager config for crc      
19:53:41 INFO  Writing dnsmasq config for crc        
```

This is because of recent PR merge from @cfergeau, this is not the good UX for `crc setup`.

**Comment by cfergeau** [comment]
Nope, it's caused by commit f70e40f45ff6a, something like this fixes the issue:
```
diff --git a/pkg/crc/preflight/preflight_checks_linux.go b/pkg/crc/preflight/preflight_checks_linux.go
index 601af77..e7de588 100644
--- a/pkg/crc/preflight/preflight_checks_linux.go
+++ b/pkg/crc/preflight/preflight_checks_linux.go
@@ -2,6 +2,7 @@ package preflight
 
 import (
 	"bytes"
+	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
@@ -15,7 +16,7 @@ import (
 	"text/template"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
-	"github.com/code-ready/crc/pkg/crc/errors"
+	crcerr "github.com/code-ready/crc/pkg/crc/errors"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/machine/libvirt"
 	"github.com/code-ready/crc/pkg/crc/oc"
@@ -175,7 +176,7 @@ func checkUserPartOfLibvirtGroup() (bool, error) {
 		logging.Debug("Current user is already in the libvirt group")
 		return true, nil
 	}
-	return false, errors.NewF("%s not part of libvirtd group", currentUser.Username)
+	return false, crcerr.NewF("%s not part of libvirtd group", currentUser.Username)
 }
 
 func fixUserPartOfLibvirtGroup() (bool, error) {
```
I'll file a PR tomorrow if noone beats me to it.

**Comment by anjannath** [comment]
Ohh yes, i was also thinking it was a side effect of the debug logs PR, but since the import of `errors` got changed, we are seeing those, so if we use the `pkg/crc/errors` package then forming a new error using `errors.New()` prints the message using `logging.Error()`.

https://github.com/code-ready/crc/blob/2780967e54f1a797706a8504938e27d1442971fd/pkg/crc/errors/error.go#L10-L18

**Comment by anjannath** [comment]
Closed via #191
